### PR TITLE
sdk 0.44.0 and golang 1.17

### PR DESCRIPTION
**Pull request by faddat**
SDK 0.43.0 no bueno.  

Also updated to the latest go because I really like what it does with go.mo﻿

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -28,6 +28,114 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```
Comment (jackzampolin) [inline go.mod:31]: Is this needed @faddat ?

**Comment by jtieri** [comment]
@faddat Can you get this PR back to your original commits without Jacks branch being included? That is part of a much larger refactor that would be messy to include here.

**Comment by faddat** [comment]
totally, sorry about the delay :D

**Comment by faddat** [comment]
this will never be super clean so I am closing it and PRing from a feature branch.

**Pull request closed without merging**